webpack.config.js

```diff
@@ -51,7 +52,11 @@ module.exports = {
     new webpack.EnvironmentPlugin({
       TRAVIS_COMMIT: 'development'
     }),
-    new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en/)
+    new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /blah/),
+    new BundleAnalyzerPlugin({
+      analyzerMode: env.analyze ? 'static' : 'disabled',
```

Comment:
You didn't like the `server` mode? Guess I've never looked at the config options. Should be good.